locale/en.po

```diff
@@ -3892,6 +3892,10 @@ msgstr "Models"
 msgid "MODIFY"
 msgstr "Modify"
 
+#: ../src/microbe_stage/editor/tooltips/SelectionMenuToolTip.tscn:163
+msgid "MODIFY_ONCE_PLACED"
+msgstr "Modify once placed"
```

Comment:
Is it the best that this text tells the player "hey, modify this" instead of being like "Can be modified after placing"? I feel like that's a bit off in terms of tone in a general characteristics info box listing.